src/ListWithoutDouble.java

```diff
@@ -0,0 +1,34 @@
+/* Написать функцию, которая убирает одинаковые значения из списка */
+
+import java.util.ArrayList;
+
+public class ListWithoutDouble {
+
+    public static ArrayList<Integer> delDouble(ArrayList<Integer> arr) {
+        for (int i = 0; i < arr.size(); i++) {
+            for(int j = i + 1; j < arr.size(); j++){
+                if (arr.get(i) == arr.get(j)){
```

Comment:
[ERROR] /Users/geoiva/Desktop/SberJava/students/maxim_rusalev/HomeWork/src/ListWithoutDouble.java:10:46: '{' is not preceded with whitespace. [WhitespaceAround]
[ERROR] /Users/geoiva/Desktop/SberJava/students/maxim_rusalev/HomeWork/src/ListWithoutDouble.java:10:46: WhitespaceAround: '{' is not preceded with whitespace. [WhitespaceAround]